volition.sln.DotSettings.user

```diff
@@ -0,0 +1,6 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
oops, I don tthink this file was supposed to be checked in.

Reply:
I don't think this is ignored, should we added this to ignore?

Reply:
you tell me!

Reply:
I believe I did add it afterwards, I will double check